### PR TITLE
f-header@2.0.0-beta.22-update-hamburger-style

### PR DESCRIPTION
**Pull request by smilburn1990**
### Added
 - new styles to the `.c-nav-toggle` class

### CODE REVIEW

**Review by simonsandersje — APPROVED** [review]

File: packages/f-header/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v2.0.0-beta-22
```
Comment (DamianMullins) [inline packages/f-header/CHANGELOG.md:7]: f-header@v2.0.0-beta.22 is already in this PR! — https://github.com/justeat/fozzie-components/pull/87

Comment (DamianMullins) [reply packages/f-header/CHANGELOG.md:7]: Can you pull that change into your maybe so we can avoid one if us having to rebase?

Comment (smilburn1990) [reply packages/f-header/CHANGELOG.md:7]: Or i could just add my changes to that PR? @DamianMullins

Comment (ashleynolan) [reply packages/f-header/CHANGELOG.md:7]: Looks like that PR has been merged now, so this will need to be `f-header@v2.0.0-beta.23`

**Comment by ashleynolan** [comment]
Looks like this got merged with #87 – closing

**Pull request closed without merging**